### PR TITLE
added harcode for markup disable

### PR DESCRIPTION
**Pull request by hasmikharutyunyan**

### CODE REVIEW

File: Api/Services/Agents/CounterpartyService.cs [hunk 1]
```diff
@@ -25,6 +27,9 @@ public class CounterpartyService : ICounterpartyService
 
         public async Task<Result<CounterpartyInfo>> Add(CounterpartyEditRequest counterparty)
         {
+            // TODO: Remove
```
Comment (mingazhev) [inline Api/Services/Agents/CounterpartyService.cs:30]: Please add the tech debt task for removal

Comment (mingazhev) [reply Api/Services/Agents/CounterpartyService.cs:30]: And this could be closer to usage

File: Api/Services/Agents/CounterpartyService.cs [hunk 2]
```diff
@@ -59,8 +64,24 @@ public async Task<Result<CounterpartyInfo>> Add(CounterpartyEditRequest counterp
                 ParentId = null,
             };
             _context.Agencies.Add(defaultAgency);
-
             await _context.SaveChangesAsync();
+
+            //TODO: Remove
```
Comment (mingazhev) [inline Api/Services/Agents/CounterpartyService.cs:69]: Need a task and longer description.
Something like "Temporary solution for CIS countries, registered before date"

**Comment by kirillta** [comment]
The function is needed on Production tomorrow morning, but this PR will not on Production tomorrow morning. Pointless

**Pull request closed without merging**